lib/cli/src/commands/run/wasi.rs

```diff
@@ -25,22 +26,28 @@ pub struct Wasi {
     #[cfg(feature = "experimental-io-devices")]
     #[clap(long = "enable-experimental-io-devices")]
     enable_experimental_io_devices: bool,
+
+    #[clap(long = "allow-multiple-wasi-versions")]
+    pub allow_multiple_wasi_versions: bool,
```

Comment:
Where is writen the documentation for those options?

Reply:
Good call, forgot that